### DISCUSSION TITLE
Identifying missing/deleted entries

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#10927] [General]
I'm using the entires fieldtype, to display links to related entries,  but over time entries must have been deleted or moved. The result is I can see missing linked entries in the fieldtype, but I've no way of knowing what the entry was?

I can see the missing ID, highlighted in red, but when I search for the ID it doesn't exist.

<img width="915" alt="Screenshot 2024-10-10 at 10 21 09" src="https://github.com/user-attachments/assets/0b88a73e-f1b7-4fe5-b133-df160b36f4e6">

Statamic retains the missing ID, but could it also retain (and display) the title?

It'd be useful to know what the missing entry is

**Comment by mscruse** [comment]
The reason is that your entries field only stores the ID of the related entry, not the entire content, so when you delete the entry it no longer has access to the title of it is so shows the ID.

**Reply by stephenmeehanuk** [reply]
Thanks, I understand. However, Statamic does retain the ID. I wouldn't expect the entire content to be stored after an entry is deleted. But, it would be handy to store the title.

Perhaps an optional config to retain the title on deletion?

**Comment by jasonvarga** [comment]
When you delete an entry, it won't remove the ID from any relationship fields. When Statamic encounters a now-invalid ID in a relationship field, it will simply ignore it.

If we were to "fix" this, the fix would be that the IDs get removed. I don't see any situation where we'd keep just a title.
